library/init/meta/lean_parser.lean

```diff
@@ -0,0 +1,44 @@
+/-
+Copyright (c) 2017 Microsoft Corporation. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sebastian Ullrich
+-/
+prelude
+import init.category.monad init.category.alternative init.meta.pexpr
+
+meta constant lean_parser : Type → Type
+
```

Comment:
Should we have a namespace called `lean`, and move `parser`, `expr`, `level`, `environment`, ... into this namespace? Right now, `expr`, `level`, ... are all top level declarations, this may be annoying in the future. Another option is to have the namespace `std`, and put the whole standard library there.